cookbook/deployment/cluster/config_flyte_deploy.rst

```diff
@@ -0,0 +1,181 @@
+#################################
+Configuring Your Flyte Deployment
+#################################
+
+As the complexity of your user base grows, you may find yourself tweaking resource assignments based on specific projects, domains, and workflows. This document walks through how and in what ways you can configure your Flyte deployment.
+
+.. _config_resource_types:
+
+***************************
+Configurable Resource Types
+***************************
+Flyte allows these custom settings along with the following combination of dimensions:
+
+- domain
+- project and domain
+- project, domain, and name (must be either the name of a workflow name or a launch plan)
+
+Please see the `Control Plane <https://docs.flyte.org/en/latest/concepts/control_plane.html>`__ document to get to know about projects and domains. 
+Along these dimensions, the following settings are configurable. 
+
+.. admonition:: Alert
+    Not all three of the combinations mentioned above are valid for each of these settings.
+
+- Defaults are used for task resource requests and limits (when not specified by the author of the task).
+- Settings for project-namespaced cluster resource configuration that feeds into admin's cluster resource manager.
+- Execution queues that are used for dynamic workflows. Effectively, they're meant to be used with constructs like AWS Batch.
+
+.. note::
+    Execution queues are used to determine where tasks yielded by a :py:func:`flytekit:flytekit.dynamic` workflow or ``map task`` run.
+
+- Determining how workflow executions are assigned to clusters in a multi-cluster Flyte deployment.
+
+.. tip::
+  The proto definition is the definitive source encapsulating which
+  `matchable attributes <https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/matchable_resource.proto>`_
+  can be customized.
+
+Each of the four above settings is discussed below. 
+
+Task Resources
+==============
+Configuring task resources includes setting default values for the requests and limits for the following resources:
+
+- cpu
+- gpu
+- memory
+- storage
+
+In the absence of an override, the global
+`default values <https://github.com/flyteorg/flyteadmin/blob/6a64f00315f8ffeb0472ae96cbc2031b338c5840/flyteadmin_config.yaml#L124,L134>`__
+in the FlyteAdmin config are used.
+
+The override values from the database are assigned at execution time.
+
+To update individual project-domain attributes, use the following command for your reference.
+
+.. prompt:: bash
+
+    curl --request PUT 'https://flyte.company.net/api/v1/project_domain_attributes/projectname/staging' \
```

Comment:
@pmahindrakar-oss @katrogan there's a flytectl thing for this now right?

Reply:
Yes . following link shows the details using flytectl 
https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_update_task-resource-attribute.html

Reply:
Similar  to this one for cluster resource attribute https://github.com/flyteorg/flytesnacks/pull/225/files#diff-9562e1e7a444fa51c89467f9756317c0990a2b2830a67a7617381f941b583a86R79-R87